Update to last VRS - Adding Aria Gen2 support

SeaOtocinclus opened the pull request:
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [s] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

SeaOtocinclus committed:
Try to use upstream vrs

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

SeaOtocinclus committed:
Fix variable usage

SeaOtocinclus committed:
Add some newly required dependencies

SeaOtocinclus committed:
enable long path for git on windows

SeaOtocinclus committed:
Remove some linux variant
- updated code is no longer directly compatible with those.

SeaOtocinclus commented:
Since we removed some linux variant -- we need to rerender

@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.23.15.25.2

SeaOtocinclus committed:
add xprs build to the loop

SeaOtocinclus committed:
Fixing variable path

SeaOtocinclus committed:
Test

SeaOtocinclus committed:
Link to ffmpeg lgpl
Do not compile with XPRS support on Windows -- not fully supported yet

SeaOtocinclus commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.24.23.37.0

SeaOtocinclus committed:
Fix for windows

SeaOtocinclus committed:
Remove mac Intel
xprsDecoder.cpp:164:24: error: use of undeclared identifier 'kCVPixelFormatType_422YpCbCr8BiPlanarFullRange'; did you mean 'kCVPixelFormatType_420YpCbCr8BiPlanarFullRange'?
  164 |         pixelFormat == kCVPixelFormatType_422YpCbCr8BiPlanarFullRange) {
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                        kCVPixelFormatType_420YpCbCr8BiPlanarFullRange

SeaOtocinclus commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.24.23.37.0

SeaOtocinclus commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2026.01.12.11.27.0

SeaOtocinclus committed:
Updated conda_build_config.yaml to require macOS 13.0+ for both the C stdlib version and deployment target since VRS source code now uses CoreVideo pixel format constants that were introduced in macOS 13.0 (Ventura):
- kCVPixelFormatType_444YpCbCr8BiPlanarVideoRange
- kCVPixelFormatType_444YpCbCr8BiPlanarFullRange

SeaOtocinclus committed:
Update deps (fmt, vrs, lz4)

SeaOtocinclus committed:
tested pixel SDK dedection

SeaOtocinclus committed:
Merge branch 'vrs_main' of github.com:SeaOtocinclus/vrs-feedstock into vrs_main

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ You are setting `MACOSX_SDK_VERSION` below `c_stdlib_version`, in conda_build_config.yaml which is not possible! Please ensure `MACOSX_SDK_VERSION` is at least `c_stdlib_version` (you can leave it out if it is equal).
If you are not setting `c_stdlib_version` yourself, this means you are requesting a version below the current global baseline in conda-forge (10.13). If this is the intention, you also need to override `c_stdlib_version` and `MACOSX_DEPLOYMENT_TARGET` locally.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/20967227403](https://github.com/conda-forge/conda-forge-webservices/actions/runs/20967227403). Examine the logs at this URL for more detail.</sub>

SeaOtocinclus committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2026.01.13.03.10.20

Other tools:
- conda-build 25.11.1
- rattler-build 0.55.1
- rattler-build-conda-compat 1.4.10

SeaOtocinclus committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2026.01.13.03.10.20

Other tools:
- conda-build 25.11.1
- rattler-build 0.55.1
- rattler-build-conda-compat 1.4.10

SeaOtocinclus committed:
fixing linting

SeaOtocinclus committed:
testing

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

SeaOtocinclus committed:
Taking back vrs repo
